lte/gateway/c/core/oai/test/mme_app_task/test_mme_app_emm_encode_decode.cpp

```diff
@@ -648,5 +734,59 @@ TEST_F(EMMEncodeDecodeTest, TestEncodeDecodeExtendedServiceRequest) {
       &original_msg.mtmsi, &decoded_msg.mtmsi, sizeof(original_msg.mtmsi)));
 }
 
+TEST_F(EMMEncodeDecodeTest, TestEncodeDecodeAuthenticationResponse) {
```

Comment:
test name is mixed up with the next test?

Reply:
resolved